examples/cgx_component_example.cgx

```diff
@@ -0,0 +1,41 @@
+<template>
+  <QMainWindow title="JSX-like syntax example">
+    <QWidget :layout_direction="QBoxLayout.Direction.TopToBottom">
+      <QLabel v-bind:text="state['title']" />
+      <QLabel v-if="content" :text="props['content']" />
+      <QLabel v-else-if="other_content" :text="other_content" />
+      <QLabel v-else text="Nothing to see here" />
+      <QLineEdit v-model="line" />
+      <QWidget :layout_direction="QBoxLayout.Direction.LeftToRight">
+        <QCheckBox v-on:state-changed="check_action" text="Click it" />
+        <QLabel text="You know you want it!" />
+      </QWidget>
+      <QPushButton text="Push me" @clicked="click_action" />
+    </QWidget>
+  </QMainWindow>
+</template>
+
+<code lang="python">
```

Comment:
is the `lang="python"` necessary for something specific?

Reply:
Could be of use for syntax highlighting in editors, but I'm not sure yet. I'm going to rename the `<code>` to `<script>` anyways.

Reply:
Are there any existing tools that know how to parse files like this and lint or syntax highlight them?

Reply:
<img width="233" alt="image" src="https://user-images.githubusercontent.com/1000968/165983237-75b66aea-d373-43b3-9609-9aec5a57c39c.png">
Seems that the vue-syntax-highlight package for Sublime uses `lang=` for checking whether the contents are coffee/typescript or otherwise just plain javascript. Quickly hacking the syntax file together with `lang="python"` results in python syntax highlighting 🎉 Apparently, as can be seen by the green highlight of `import`, the linter plugin + flake8 + black are confused 😅